portfolio/src/main/webapp/index.html

```diff
@@ -65,20 +66,23 @@ <h1>Adam's Personal Portfolio</h1>
 
       <p>Click <a href="random-pair.html"><button type="button">here</button>
       </a> to see a random pairing of Luna and Food!</p>
-    </div>
-    <div id="content">
```

Comment:
No action required, just a general thought. It's often recommended to use <div> tags to wrap related content. This helps the structure of the html stay in sync with the logical structure of your page. eg: all gallery related content is under the same dom element. Div tags don't affect the layout of the page by default but you can use css to apply the effects you want, eg: setting margin-top to create visual space between sections in your doc.

I don't think you need to make these changes now, but something to keep in mind for more complex html projects.